### DISCUSSION TITLE
Am i the only one that having trouble with subtitles ?

### DISCUSSION THREAD

**Discussion by sashalpb** [iina/iina#5195] [General]
Thats weird but everything was perfect before, like i did not had to load the subtitle manually each times, it would be done automatically but now it never works not a single time, but i haven't change anything, maybe i changed something without knowing but let me know if its on my side bc thats kinda annoying, love the app btw great job guys !

**Comment by low-batt** [comment]
First step is to check to match sure the feature that automatically loads subtitles is enabled:
- Start IINA
- Click on  `Settings…` under the `IINA` menu
- The settings panel appears
- On the left side of the panel click on `Subtitle`
- On the right side of the panel at the top, confirm the `Auto Load` setting is not set to `Disabled`
